game-core/src/main/java/games/strategy/engine/data/Territory.java

```diff
@@ -9,12 +9,13 @@
  */
 public class Territory extends NamedAttachable implements NamedUnitHolder, Comparable<Territory> {
   private static final long serialVersionUID = -6390555051736721082L;
-  private final boolean m_water;
-  private PlayerID m_owner = PlayerID.NULL_PLAYERID;
-  private final UnitCollection m_units;
+
+  private final boolean water;
+  private PlayerID owner = PlayerID.NULL_PLAYERID;
+  private final UnitCollection units;
   // In a grid-based game, stores the coordinate of the Territory
   @SuppressWarnings("unused")
-  private final int[] m_coordinate;
+  private final int[] coordinate;
```

Comment:
Instead of Renaming, why not just remove this variable?

Reply:
Sorry for not noting that...  I actually did remove it originally, but it requires making some changes over in `GameParser`, so I decided not to mix concerns in this "Checkstyle" PR.  I will follow up in a separate PR, just in case it gets out-of-hand. :smile: